personal_profile/tests.py

```diff
@@ -31,13 +31,30 @@ def profile_1(db, user_1):
     return profile_1
 
 
+@pytest.mark.django_db
 class TestProfileModel:
-    # Testing different Personal Profile attributes
+    # Testing different Personal Profile saving and deletation
 
-    def test_newprofile(self, profile_1, user_1):
+    def test_save_newprofile(self, profile_1, user_1):
```

Comment:
please rename to clear name test_new_profile, 
tests should tell what they are testing

Please do for the other tests

Reply:
done